bin/anvi-compute-functional-enrichment

```diff
@@ -1,143 +1,31 @@
 #!/usr/bin/env python
 # -*- coding: utf-8
-"""A program that computes enrichment scores and group associations for annotated
-   entities across groups of genomes or samples"""
 
 import sys
 
-import anvio
-import anvio.kegg as kegg
-import anvio.terminal as terminal
-import anvio.summarizer as summarizer
-import anvio.filesnpaths as filesnpaths
-import anvio.genomedescriptions as genomedescriptions
+from anvio.errors import ConfigError
+from anvio.terminal import Run
 
-from anvio.errors import ConfigError, FilesNPathsError
+__description__ = ("A program that is no more. Read on")
+__provides__ = ['functional-enrichment-txt']
 
-__author__ = "Developers of anvi'o (see AUTHORS.txt)"
-__copyright__ = "Copyleft 2015-2018, the Meren Lab (http://merenlab.org/)"
-__credits__ = []
-__license__ = "GPL 3.0"
-__version__ = anvio.__version__
-__maintainer__ = "Iva Veseli"
-__email__ = "iveseli@uchicago.edu"
-__resources__ = [("A description of the enrichment script run by this program can be found in Shaiber et al 2020", "https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02195-w"),
-                  ("An example of pangenome functional enrichment in the context of the Prochlorococcus metapangenome from Delmont and Eren 2018 is included in the pangenomics tutorial", "http://merenlab.org/2016/11/08/pangenomics-v2/")]
-__tags__ = ["pangenomics", "functions", "metabolism"]
-__requires__ = ['kegg-metabolism', 'groups-txt', 'misc-data-layers', 'pan-db', 'genomes-storage-db', 'external-genomes', 'internal-genomes', 'functions']
-__provides__ = ['functional-enrichment-txt',]
-__description__ = ("This is a driver program for `anvi-script-enrichment-stats`, a script that computes enrichment scores "
-                   "and group associations for annotated entities (ie, functions, KEGG Modules) across groups of genomes or "
-                   "samples.")
+def main():
+    run = Run()
+    run.warning("This program is broken into multiple programs. Our VERY FAVORITE one is "
+                "the metabolic enrichment one, but you feel free to choose the one from "
+                "the list of minions below based on what you wish to do:",
+                header="THIS PROGRAM IS NO MORE", lc="yellow")
 
```

Comment:
😂 
anvi'o should really not be picking favorites from among her children, but I cannot say I disagree with her choice here :p